conf/train/decision_tree_gpt2.yml

```diff
@@ -22,4 +23,4 @@ steps: 500000
 log_freq: 100
 checkpoint_freq: 5000
 model: *std_gpt2
-baseline_models: [*least_squares, *knn_3, *xgboost]
+baseline_models: [*knn_3, *xgboost, *decision_tree, *decision_tree_4]
```

Comment:
Does `max_depth=None` for the decision tree let it "fit" until all points have a leaf? I'm not super familiar with sklearn in this regard

Reply:
yeah or until you have less than the minimum required for a split